doc/changes/added/12538.md

```diff
@@ -1,3 +1,4 @@
 - Add a `(conflict error|ignore)` option to the cram stanza. When `(conflict
   error)` is set, the cram test will fail in the presence of conflict markers.
-  (#12538, fixes #12512, @rgrinberg)
+  Git, diff3 and jujutsu conflicts are supported.
```

Comment:
How about just removing this sentence and combining the list of issues, pr's and contributors?

And instead of

> Git, diff3 and jujutsu conflicts are supported.

We can have:

> Git, diff3, and jujutsu conflict markers are detected.